village/src/components/SmurfForm.js

```diff
@@ -1,10 +1,11 @@
 import React, { Component } from 'react';
-
+import axios from 'axios';
 class SmurfForm extends Component {
 
   constructor(props) {
     super(props);
     this.state = {
+      smurfs: [],
```

Comment:
There's no reason to keep a `smurfs` array here since you're not even rendering that data any where in this component